# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../faad/noplugins.patch.faad2
# Copyright (C) 2006 - 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- faad2/configure.in.orig	2006-09-04 15:32:34.000000000 +0000
+++ faad2/configure.in	2006-09-04 15:58:06.000000000 +0000
@@ -118,6 +118,9 @@
   AC_DEFINE([HAVE_BMP], 1, [User wants beep media player plugin built])
   AM_CONDITIONAL([HAVE_XMMS], true)
   AM_CONDITIONAL([HAVE_BMP], true)
+else
+  AM_CONDITIONAL([HAVE_XMMS], false)
+  AM_CONDITIONAL([HAVE_BMP], false)
 fi
 
 if test x$WITHDRM = xyes; then
@@ -128,10 +131,10 @@
 AC_CONFIG_FILES(libfaad/Makefile)
 AC_CONFIG_FILES(common/Makefile)
 AC_CONFIG_FILES(common/mp4ff/Makefile)
-AC_CONFIG_FILES(plugins/Makefile)
-AC_CONFIG_FILES(plugins/xmms/Makefile)
-AC_CONFIG_FILES(plugins/xmms/src/Makefile)
-AC_CONFIG_FILES(plugins/mpeg4ip/Makefile)
+#AC_CONFIG_FILES(plugins/Makefile)
+#AC_CONFIG_FILES(plugins/xmms/Makefile)
+#AC_CONFIG_FILES(plugins/xmms/src/Makefile)
+#AC_CONFIG_FILES(plugins/mpeg4ip/Makefile)
 AC_CONFIG_FILES(faad2.spec)
 AC_CONFIG_FILES(frontend/Makefile)
 AC_CONFIG_FILES(Makefile)
--- faad2/Makefile.am.orig	2004-07-27 10:52:20.000000000 +0200
+++ faad2/Makefile.am	2006-09-04 18:14:20.000000000 +0200
@@ -1,4 +1,4 @@
-SUBDIRS = libfaad common frontend plugins
+SUBDIRS = libfaad common frontend
 
 EXTRA_DIST = faad2.spec
 
